src/core/stream.c

```diff
@@ -37,6 +37,12 @@ QuicStreamInitialize(
         Status = QUIC_STATUS_OUT_OF_MEMORY;
         goto Exit;
     }
+
+    QuicTraceEvent(
```

Comment:
If we're willing to break the ETW "contract" then we can move StreamOpen event here, and make a new StreamStarted event where StreamOpen is now.

Reply:
I don't think we want to break that. We have actively shipped code using the existing ETW manifest.